src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const imgList = document.querySelector('#thumbs');
+
+imgList.addEventListener('click', (action) => {
+  action.preventDefault();
+
+  const imgLink = action.target.closest('.list-item__link');
+
```

Comment:
```suggestion
  const imgLink = action.target.closest('.list-item__link');
```
зроби перевірку на випадок якщо closest не знайде елемент